mkt/reviewers/views.py

```diff
@@ -1213,3 +1214,50 @@ class ReviewerScoreViewSet(CORSMixin, MarketplaceView, viewsets.ModelViewSet):
     # queryset to prevent instances with other note_key values from ever being
     # returned.
     queryset = ReviewerScore.objects.filter(note_key=mkt.REVIEWED_MANUAL)
+
+
+@reviewer_required
+def moderatelog(request):
+    form = ModerateLogForm(request.GET)
+    modlog = ActivityLog.objects.editor_events()
+    if form.is_valid():
+        if form.cleaned_data['start']:
+            modlog = modlog.filter(created__gte=form.cleaned_data['start'])
+        if form.cleaned_data['end']:
+            modlog = modlog.filter(created__lt=form.cleaned_data['end'])
+        if form.cleaned_data['search']:
+            modlog = modlog.filter(action=form.cleaned_data['search'].id)
+
+    pager = paginate(request, modlog, 50)
+    data = context(request, form=form, pager=pager, tab='moderatelog')
+    return render(request, 'reviewers/moderatelog.html', data)
+
+
+@reviewer_required
+def moderatelog_detail(request, id):
```

Comment:
nit: `id` is a built-in function, so I like to avoid using it as a variable/argument name when possible